user_docs/en/userGuide.t2t

```diff
@@ -174,7 +174,7 @@ The third lets you control if this Welcome dialog should appear each time NVDA s
 +++ Data usage statistics dialog +++[UsageStatsDialog]
 Starting from NVDA 2018.3, the user is asked if they want to allow usage data to be sent to NV Access in order to help improve NVDA in the future. 
 When starting NVDA for the first time, a dialog will appear which will ask you if you want to accept sending data to NV Access while using NVDA.
-You can read more info about the data gathered by NV Access in Chapter 11 below.
+You can read more info about the data gathered by NV Access in [this section #GeneralSettingsGatherUsageStats].

```

Comment:
NIT: Make the link text perceivable by someone trying to use links list, links navigation, speech recognition, etc. For example, "the section on privacy" or "You can learn more about privacy in chapter n" ...  etc.